Packs/Nexthink/Integrations/Nexthink/Nexthink.py

```diff
@@ -38,6 +39,15 @@
 TEST_MODULE = "(select (name) (from device ) (limit 1))"
 
 
+def is_valid_hostname(hostname):
```

Comment:
Hi,
 According to the product documentation https://doc.nexthink.com/Documentation/Nexthink/latest/APIAndIntegrations/NXQLDataModel#device
the field name in data model device is not a full domain name (i.e.  hostname.domain) 
![image](https://user-images.githubusercontent.com/2477859/135486068-55952822-5c70-4876-bffe-94798e5c64a9.png)
so you do not need to do the split/parsing of the hostname.

Reply:
Yes, correct but we still need to validate if the user have provided the correct name.

Reply:
I have modified the code to address this observation :-)